src/Controls/src/Core/Handlers/Items/iOS/ItemsViewLayout.cs

```diff
@@ -595,5 +604,51 @@ internal void ClearCellSizeCache()
 		{
 			_cellSizeCache.Clear();
 		}
+
+		CGSize TryFindEstimatedSize(CGSize existingMeasurement)
+		{
+			if (CollectionView == null || GetPrototypeForIndexPath == null)
+				return existingMeasurement;
+
+			//Since this issue only seems to be reproducible on Horizontal scrolling, we only check for that
+			if (ScrollDirection == UICollectionViewScrollDirection.Horizontal)
+			{
+				return FindEstimatedSizeUsingWidth(existingMeasurement);
+			}
+
+			return existingMeasurement;
+		}
+
+		CGSize FindEstimatedSizeUsingWidth(CGSize existingMeasurement)
+		{
+			// TODO: Handle grouping
```

Comment:
Do we need a separate issue for this or is this no longer an issue?

Reply:
Yes done https://github.com/dotnet/maui/issues/20654